frontend/src/app/main/ui.cljs

```diff
@@ -28,7 +28,7 @@
 (mf/defc on-main-error
   [{:keys [error] :as props}]
   (mf/use-effect (st/emitf (rt/assign-exception error)))
-  [:span "Internal application errror"])
+  [:span "Internal application error"])
```

Comment:
user visible?

Reply:
Noup, this really a place holder, so it is ok as it is.